core/src/main/java/org/hibernate/ogm/exception/spi/ErrorHandlingStrategy.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.exception.spi;
+
+
+/**
+ * @author Gunnar Morling
+ *
+ */
+public enum ErrorHandlingStrategy {
+	ABORT, CONTINUE;
```

Comment:
Right, it'd be simpler to use. You wouldn't have to implement that in each handler. And we could provide more advanced strategies such as exponential back-off etc.